src/blockrelay/graphene.cpp

```diff
@@ -1236,6 +1309,11 @@ void SendGrapheneBlock(CBlockRef pblock, CNode *pfrom, const CInv &inv, const CM
             CGrapheneBlock grapheneBlock(pblock, mempoolinfo.nTx, nSenderMempoolPlusBlock,
                 NegotiateGrapheneVersion(pfrom), NegotiateFastFilterSupport(pfrom));
 
+            {
+                LOCK(cs_graphenerecovery);
+                grapheneRecoveryBlock[pfrom->GetId()] = grapheneBlock;
+            }
```

Comment:
There seems to be no purpose for storing these grapheneBlocks's...you store them but never used them anywhere AFAICT?

Reply:
Shoot you're right. I don't remember exactly why this was in here to begin with, but I think it was a first attempt at the receiver keeping track of graphene blocks before I ultimately moved the logic to `mapGrapheneSentBlocks` in blockrelay_common.cpp.

Reply:
Removed grapheneRcoveryBlock in commit 4af195d.